src/main/java/shop/service/impl/UserServiceImpl.java

```diff
@@ -1,4 +1,38 @@
 package shop.service.impl;
 
-public class UserServiceImpl {
-}
+import shop.dao.UserDAO;
+import shop.dao.impl.UserDAOImpl;
+import shop.entity.User;
+import shop.enums.UserRole;
+import shop.service.UserService;
+
+import java.util.UUID;
+
+public class UserServiceImpl implements UserService {
+    private UserDAO userDAO = new UserDAOImpl();
+
+    @Override
+    public boolean login(String email, String password) {
+        return isExist(email);
```

Comment:
here u should call findUser(String email) and if present -   match the password . After that, you return true/false if password matches